recipes/hypsoreader/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "HYPSO-Reading" %}{% set version = "0.1.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/DevAlvaroF/HYPSO-Reading/archive/v0.1.0.tar.gz
+  sha256: e8470d3e0cde767182f3df45c88c15abf561e07c5c1d2f38d9698a0fe4a28e01
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python =3.9
+    - pip
+    - setuptools >=38.2.5
+    - setuptools-scm
+  run:
+    - python =3.9
```

Comment:
Same as in https://github.com/conda-forge/staged-recipes/pull/23791/files#r1306025206